src/pretalx/agenda/views/sneakpeek.py

```diff
@@ -0,0 +1,16 @@
+from django.views.generic import TemplateView
+
+from pretalx.common.mixins.views import PermissionRequired
+
+
+class SneakpeekView(PermissionRequired, TemplateView):
+    template_name = 'agenda/sneakpeek.html'
+    permission_required = 'agenda.view_sneak_peek'
+
+    def get_permission_object(self):
+        return self.request.event
```

Comment:
I think it would be neat to redirect to the actual agenda once it exists and is visible. (You could do that in ``dispatch``).

Reply:
done. :)